templates/ceph/index.html

```diff
@@ -296,15 +296,16 @@ <h3 class="p-heading--4">Managed Ceph</h3>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Fully managed storage cluster, on your servers, at your location</li>
         <li class="p-list__item is-ticked">Built in 2 weeks using a proven reference architecture</li>
-        <li class="p-list__item is-ticked">Customise the architecture in iterations based on your storage needs</li>
+        <li class="p-list__item is-ticked">Customize the architecture in iterations based on your storage needs</li>
         <li class="p-list__item is-ticked">Your team has access to all machines at all times</li>
         <li class="p-list__item is-ticked">Modern monitoring and log management</li>
         <li class="p-list__item is-ticked">Canonical hands over the keys when you are ready</li>
       </ul>
-      <p><a href="/ceph/managed">Find out more&nbsp;&rsaquo;</a></p>
+      <p><a href="/ceph/managed">Learn more about fully managed Ceph&nbsp;&rsaquo;</a></p>
     </div>
   </div>
 </section>
+
 <section class="p-strip--light is-deep is-bordered">
   {% include "shared/_resources_ceph.html" %}
```

Comment:
The copy here looks a bit different on the doc, left a couple comments

Reply:
Good catch! As this is a section that will be rebranded very shortly, I am comfortable leaving it as is. I will make sure not to accept those changes